vars/longevityPipeline.groovy

```diff
@@ -51,6 +170,11 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('email_recipients', 'qa@scylladb.com')}",
                    description: 'email recipients of email report',
                    name: 'email_recipients')
+
+            string(defaultValue: "${pipelineParams.get('test_config', '')}",
```

Comment:
can we not expose this value ?
also seems like `test_name` isn't part of params

Reply:
We definitely need it in functions and therefore had to expose it, also we are currently exposing it via pipelineParam.

Reply:
@fruch, I will add test_name too.

Reply:
@fruch , test_name added